text/0000-bee-readme.md

```diff
@@ -0,0 +1,114 @@
++ Feature name: `bee-readme`
++ Start date: 2019-09-24
++ RFC PR: [iotaledger/bee-rfcs#0000](https://github.com/iotaledger/bee-rfcs/pull/0000)
++ Bee issue: [iotaledger/bee#0000](https://github.com/iotaledger/bee/issues/0000)
+
+# Summary
+
+This RFC introduces a README for the Bee project, explaining that it will be
+a framework written in Rust allowing to write nodes participating in the IOTA
+network and tools interacting with it.
+
+# Motivation
+
+In order to motivate our own work and explain to the community what we are doing,
+it is crucial to have a good README.
+
+# Detailed design
+
+The README is available at
+[`SuperFluffy/bee-readme/README.md`](https://github.com/SuperFluffy/bee-readme/blob/master/README.md)
+and reproduced below:
+
+> # The IOTA Bee framework
+> 
+> Bee is a Rust framework to build IOTA nodes and tools that interact with the IOTA network. The final product is envisioned to be a highly modular collection of foundational crates that can be mixed and matched, and extended upon.
+> 
+> Bee may be seen as an effort to streamline the IOTA Foundation's different libraries into one fundamental collection of Rust crates. These will expose foreign function interfaces (FFIs) to be used in more high level languages, and form the basis for the next iteration of client APIs.
+> 
+> ## Outline
+> 
+> + [Motivation]
+> + [Milestones]
+> + [How to contribute](#contributing)
```

Comment:
Because I am very whimsical and liked it that way... For lack of a better reason, I thought it reads more naturally. You prefer to make it consistent?